result/wrap-regex.js

```diff
@@ -0,0 +1,13 @@
+function a() {
```

Comment:
__[development][eslint]__
* Error - 'a' is defined but never used (no-unused-vars)